sdk/src/wallet/core/builder.rs

```diff
@@ -212,16 +212,8 @@ where
             self.secret_manager = secret_manager;
         }
 
-        // May use a previously stored wallet alias if it wasn't provided
-        if self.alias.is_none() {
-            self.alias = loaded_wallet_builder.as_ref().and_then(|builder| builder.alias.clone());
-        }
-
-        if self.address.is_none() {
-            self.address = loaded_wallet_builder
-                .as_ref()
-                .and_then(|builder| builder.address.clone());
-        }
+        // TODO if address and/or bip path were provided we should check whether they might
+        // conflict with stored data in the wallet db.
```

Comment:
Open an issue for this?

Reply:
https://github.com/iotaledger/iota-sdk/issues/2016